.gitignore

```diff
@@ -4,6 +4,6 @@ __pycache__/
 hc.sqlite
 hc/local_settings.py
 static-collected
-
+static/
```

Comment:
We do not want to ignore the `static` directory. Rather, we want to ignore all our locally 'collected' static files which will be in `static-collected` since don't want those to go to the server.
`static-collected` has already been ignored for us and that's fine. 
As for`static`, please un-ignore it. It contains all the vital CSS and JavaScript files we need in the application. It's from here that these files while will sourced and collected into `static-collected` when when we run `python manage.py collectstatic` during the testing, staging and production deploys.